build_image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+oc create -f deploy/crds/neutron_v1_neutronovsagent_crd.yaml
+oc create -f deploy/crds/neutron_v1_neutronsriovagent_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovsnodeosps_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovncontrollers_crd.yaml
+make
+operator-sdk build --image-builder buildah "${REGISTRY:-quay.io/openstack-k8s-operators/neutron-operator:v0.0.3}"
```

Comment:
maybe we should start have a latest tag?

Reply:
Agree changed